Clarify CRON_NAME (#1006)
cirruslabs/cirrus-ci-docs: docs/guide/writing-tasks.md

CIRRUS_CHANGE_IN_REPO | Git SHA
CIRRUS_CHANGE_MESSAGE | Commit message or PR title and description, depending on trigger event (Non-PRs or PRs respectively).
CIRRUS_CHANGE_TITLE | First line of `CIRRUS_CHANGE_MESSAGE`

> CIRRUS_CRON | Name used in Cron settings if a build was triggered by the [Cron service](#cron-builds). For example, `nightly`.

Reviewer: I believe the first mention of Cron should be a link to the documentation, otherwise the user's stack gets filled up with more questions.

What do you think about:

```suggestion
CIRRUS_CRON | [Cron Build](#cron-builds) name configured in the repository settings if this build was triggered by Cron. For example, `nightly`.
```
